docs/data/data-grid/components/toolbar/toolbar.md

```diff
@@ -87,3 +86,11 @@ It implements the roving tabindex pattern.
 |                            <kbd class="key">Right</kbd> | Moves focus to the next button.          |
 |                             <kbd class="key">Home</kbd> | Moves focus to the first button.         |
 |                              <kbd class="key">End</kbd> | Moves focus to the last button.          |
+
+## Legacy Toolbar
+
+:::warning
+Deprecated API.
+:::
+
+The components above replaced the [Legacy Toolbar](/x/react-data-grid/components/#legacy-toolbar), e.g. `<GridToolbarContainer>`.
```

Comment:
```suggestion
The components above replaced the [legacy toolbar](/x/react-data-grid/components/#legacy-toolbar), e.g. `<GridToolbarContainer />`.
```

Reply:
Partially approved, `<GridToolbarContainer />` is visually heavy. We went with `<GridToolbarContainer>` in cases where the component is meant to be used with children: https://www.notion.so/mui-org/Style-conventions-for-MUI-components-and-products-dff7d2d7d1ba4a4abd47f48cf345b800?pvs=4#129cbfe7b6608048bf6ec340820fb641.